cdp4dal/src/main/java/cdp4dal/AvailableDals.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * AvailableDals.java
+ *
+ * Copyright (c) 2015-2019 RHEA System S.A.
+ *
+ * Author: Alex Vorobiev, Yevhen Ikonnykov, Sam Gerené
+ *
+ * This file is part of CDP4-SDKJ Community Edition
+ *
+ * The CDP4-SDKJ Community Edition is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * The CDP4-SDKJ Community Edition is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package cdp4dal;
+
+import cdp4dal.composition.DalMetaData;
+import cdp4dal.dal.Dal;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Getter;
+import org.apache.commons.lang3.tuple.Pair;
+
+/**
+ * Instantiated by MEF to provide a list of {@link Dal} available in the application
+ */
+// TODO revise it later whether it is applicable for Java desktop client implementation
```

Comment:
yeah, i can imagine this does not make sense for Java, these classes are very much MEF/C# oriented